### ISSUE TITLE
Add web streaming (webrtmp)

### ISSUE THREAD

**Issue by victorges** [livepeer/ui-kit#11]

**Comment by pglowacky** [comment]
Moving back to backlog because we want to do this but it's not required for launch

**Comment by victorges** [comment]
I thought we were only keeping on backlog the things we want to do on this
launch, or at least that we have decided to prioritize and "definitely" implement, which doesn't seem to be the case for this one. It will depend on multi-node and then on some investigation on how to use the mist protocols instead, likely needing to reimplement the streaming logic and maybe do some infra tunings to make it work. That will take more than 2 points and is not something we can just include in the backlog without a product/prioritization discussion. 

((IMO we should prioritize it, but we haven't made that decision)).

Unless we decide we want to include the alpha/unofficial web streaming that we have today in the SDK. Then it would be as easy as this task is estimated right now. But we chatted yesterday that we don't want that, so I moved it to icebox.
